Current-Chapters.csv

```diff
@@ -163,7 +163,7 @@ USA,New York,New York City,https://www.meetup.com/rladies-newyork/,https://twitt
 USA,North Carolina,Charlotte,"","","","",,"","","","","",Jessica Dunn,Prospective
 USA,North Carolina,RTP (Research Triangle Park),https://www.meetup.com/R-Ladies-RTP/,https://twitter.com/RLadiesRTP,rtp@rladies.org,"",,"","",https://github.com/rladies/meetup-presentations_rtp,"","","Mine Cetinkaya-Rundel, Elaine McVey",Active
 USA,Ohio,Cincinnati,https://www.meetup.com/rladies-cincinnati/,"","","",,"","","","","",Renata Gerecke,Progress
-USA,Ohio,Columbus,https://www.meetup.com/RLadies-Columbus/,https://twitter.com/RLadiesColumbus,columbus@rladies.org,"",,"","",https://github.com/rladies/meetup-presentations_columbus,"","","Ezgi Karaesmen, Amanda Gadrow, Nicole Thomas",Active
+USA,Ohio,Columbus,https://www.meetup.com/RLadies-Columbus/,https://twitter.com/RLadiesColumbus,columbus@rladies.org,"",,"","",https://github.com/rladies/meetup-presentations_columbus,"","","Mine Dogucu, , Amanda Gadrow, Ezgi Karaesmen, Katie Sasso",Active
```

Comment:
There is an extra comma in after your name. Not a big deal. :) I will fix it.
![image](https://user-images.githubusercontent.com/1934546/45315630-be6d1680-b4e9-11e8-8993-d0850584a079.png)